docs/src/views/Accessibility.tsx

```diff
@@ -0,0 +1,452 @@
+    </p>
+    <p>
+      could be implemented as separate controls, so you would navigate between them with tab. Or
+      they could be grouped together to behave as a toolbar. The first approach has advantages that
+      blind users would have awareness of all the controls available to them, but at the expense of
+      having to press tab multiple times to move around the application.
+    </p>
+    <p>
+      These types of decisions are ultimately what will make the application easy or hard for users
+      to navigate - and although Stardust make this easier to change than pure html implementations
+      - it is still costly in terms of developer time to correctly make these types of changes
+      later.
+    </p>
+    <p>
+      Our strong recommendation here is to have clear accessibility designs and example user flows
+      as early in the design process as possible.{' '}
+    </p>
+    <Header as="h1" content="Stardust basics" />
+    <p>
+      Stardust shares many of the same goals as it's predecessor - Sementic UI. It aims to enable
+      users to build up beautiful complex components and form them into applications. In additional
+      Stardust brings far greater freedom in theming, overriding behaviours / styling and far
+      greater accessibility support.
+    </p>
+    <p>
+      The main source of documentation is the live documentation at the{' '}
+      <a href="https://stardust-ui.github.io/react/">DocSite</a>.
+    </p>
+
+    <Header as="h2" content="Using DocSite" />
+    <p>
+      The <a href="https://stardust-ui.github.io/react/">DocSite</a>, among other things, contains a
+      catalogue of Stardust components. For each component, you can see:
+    </p>
+    <ul>
+      <li>The component description</li>
+      <li>
+        The default accessibility behaviour, alternative behaviours and other accessibility
+        considerations
+      </li>
+      <li>Properties</li>
+      <li>Variations with examples</li>
+    </ul>
+    <p>
+      Examples show the recommended way of using the components in the different variations. For
+      each example, you are able to edit the JSX code, see the rendered HTML, change themes and
+      validate the rendering in RTL scenario.{' '}
+    </p>
+    <p>Integration of accessibility testing into the DocSite is under consideration.</p>
```

Comment:
Stardust basics and Using DocSite probably do not need to be here since the user already is using doc site